test/pools/jemalloc_coarse_devdax.cpp

```diff
@@ -7,14 +7,39 @@
 
 #include "pool_coarse.hpp"
 
+using devdax_params_unique_handle_t =
+    std::unique_ptr<umf_devdax_memory_provider_params_t,
+                    decltype(&umfDevDaxMemoryProviderParamsDestroy)>;
+
+devdax_params_unique_handle_t create_devdax_params() {
+    char *path = getenv("UMF_TESTS_DEVDAX_PATH");
+    char *size = getenv("UMF_TESTS_DEVDAX_SIZE");
+    if (path == nullptr || path[0] == 0 || size == nullptr || size[0] == 0) {
```

Comment:
hmm... not sure if I got your comment right, but in case `devdaxParams` handle is `NULL` we return empty vector (`std::vector<poolCreateExtParams>{};`) and the test is silently passing with zero actual test cases (tested this locally). I guess we could SKIP if empty vector is passed to the `umfPoolTest` (but I'm not sure if this will work with other test suites using `umfPoolTest`)

Reply:
> in case devdaxParams handle is NULL we return empty vector (`std::vector<poolCreateExtParams>{};`) 

Ah, you are right, I worgot that in case of `NULL` we are not instantiating the test at all.

> and the test is silently passing with zero actual test cases (tested this locally)

If params is `NULL` the test is not instantiated at all with this PR.

> I guess we could SKIP if empty vector is passed to the umfPoolTest

As I understand, the test test is not instantiated if the vector is empty.

Reply:
I understand that you want somehow check if the required environment variables were not set, but I do not know what is the right way.

I think it is not a good idea: passing `NULL` params and check inside generic test if it is an instantiation of the test for devdax provider and params is `NULL` and SKIP.

Reply:
to conclude I think we have this silent "SKIP" issue in multiple places in our tests, so we don't have to fix it in this PR; let's leave it as is